neural_style.py

```diff
@@ -69,9 +85,27 @@ def build_parser():
     parser.add_argument('--learning-rate', type=float,
             dest='learning_rate', help='learning rate (default %(default)s)',
             metavar='LEARNING_RATE', default=LEARNING_RATE)
+    parser.add_argument('--beta1', type=float,
+            dest='beta1', help='Adam: beta1 parameter (default %(default)s)',
+            metavar='BETA1', default=BETA1)
+    parser.add_argument('--beta2', type=float,
+            dest='beta2', help='Adam: beta2 parameter (default %(default)s)',
+            metavar='BETA2', default=BETA2)
+    parser.add_argument('--eps', type=float,
+            dest='epsilon', help='Adam: epsilon parameter (default %(default)s)',
+            metavar='EPSILON', default=EPSILON)
     parser.add_argument('--initial',
             dest='initial', help='initial image',
             metavar='INITIAL')
+    parser.add_argument('--initial-noiseblend', type=float,
+            dest='initial_noiseblend', help='ratio of blending initial image with normalized noise (if no initial image specified, content image is used) (default %(default)s)',
+            metavar='INITIAL_NOISEBLEND')
+    parser.add_argument('--preserve-colors', type=int,
```

Comment:
Should be a boolean flag (see https://github.com/anishathalye/dotbot/blob/master/dotbot/cli.py#L15 for an example)